packages/manager/modules/telecom-universe-components/src/telecom/pack/migration/pack-migration-process.service.js

```diff
@@ -169,6 +173,32 @@ export default /* @ngInject */ function($q, OvhApiPackXdsl, Poller) {
       });
     }
 
+    // Installation type post params
+    if (migrationProcess.selectedOffer.pto) {
```

Comment:
Maybe replace switch case by a dictionnary here => will save ~20 lines

Reply:
Hello @antonymarion, would you have an example on how to do it? it's a good idea.

Reply:
You can put a new object in constant file:
```js
const MY_OBJECT = {
  [FIBER_PTO.FIBER_PTO_YES]: INSTALL_TYPE.activate,
  [FIBER_PTO.FIBER_PTO_YES_BUT_NOT_KNOWN]: INSTALL_TYPE.activate_undefined,
  [FIBER_PTO.FIBER_PTO_MULTI_OTP]: INSTALL_TYPE.multi_otp,
  DEFAULT: INSTALL_TYPE.create,
};
```
And use it on controller:

```js
if (migrationProcess.selectedOffer.pto) {
  assign(postParams, {
    installationType:
      MY_OBJECT[migrationProcess.selectedOffer.selectedPto] ||
      MY_OBJECT.DEFAULT,
  });
}
```